plugins/nodes/src/js/columns/NodesTablePublicIPColumn.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { TextCell } from "@dcos/ui-kit";
+
+import { default as schema } from "../data/NodesResolver";
+import { NodeNetwork } from "../data/NodesNetworkClient";
+import graphqlObservable from "reactive-graphql";
+import gql from "graphql-tag";
+import { map, switchMap, startWith } from "rxjs/operators";
+import { componentFromStream } from "@dcos/data-service";
+import { Observable } from "rxjs";
+import { Trans } from "@lingui/react";
+
+const getGraphQl = (privateIP: string) =>
+  graphqlObservable<{ node: { network: NodeNetwork } }>(
+    gql`
+      query {
+        node(privateIP: $privateIP) {
+          network {
+            public_ips
+          }
+        }
+      }
+    `,
+    schema,
+    { privateIP }
+  ).pipe(map(response => response.data));
+
+const Column = componentFromStream<{ privateIP: string }>(
+  (props$): Observable<React.ReactNode> => {
+    return (props$ as Observable<{ privateIP: string }>).pipe(
+      switchMap(({ privateIP }) => getGraphQl(privateIP)),
+      map(({ node }: { node: { network: { public_ips: string[] } } }) => {
+        if (
+          node.network == null ||
+          node.network.public_ips == null ||
+          node.network.public_ips.length === 0
+        ) {
+          return <TextCell>N/A</TextCell>;
```

Comment:
Should `N/A` be wrapped in `<Trans>`?

Reply:
You are absolutely right. I didn’t got into the habit yet to use trans around copy will change.